core/ledger/kvledger/tests/util.go

```diff
@@ -231,18 +230,6 @@ func setBlockFlagsToValid(block *common.Block) {
 		lutils.NewTxValidationFlagsSetValue(len(block.Data.Data), protopeer.TxValidationCode_VALID)
 }
 
-func dropCouchDBs(t *testing.T, couchdbConfig *couchdb.Config) {
-	couchInstance, err := couchdb.CreateCouchInstance(couchdbConfig, &disabled.Provider{})
-	require.NoError(t, err)
-	dbNames, err := couchInstance.RetrieveApplicationDBNames()
-	require.NoError(t, err)
-	for _, dbName := range dbNames {
-		db := &couchdb.CouchDatabase{
-			CouchInstance: couchInstance,
-			DBName:        dbName,
-		}
-		response, err := db.DropDatabase()
-		require.NoError(t, err)
-		require.True(t, response.Ok)
-	}
+func dropCouchDBs(t *testing.T, couchdbConfig *ledger.CouchDBConfig) {
+	statecouchdb.CleanupDB(t, couchdbConfig)
```

Comment:
This anyways is causing a dependency on "statecouchdb" package. Why not use the SetupCouchDB function also from that package? Derek had earlier created couchdbtest package for this purpose.